src/v2/types.ts

```diff
@@ -136,6 +137,8 @@ export type ComputedUserReserve = UserReserveData & {
   stableBorrows: string;
   stableBorrowsETH: string;
   stableBorrowsUSD: string;
+  stableBorrowAPR: string;
+  stableBorrowAPY: string;
```

Comment:
Maybe we should call it userStableBorrowAPY/APR so in the future we remember why it needs to be on the UserReserve object instead of Reserve?

Reply:
its already inside of user method so should be ok i think